python/lsst/sims/utils/AstrometryUtils.py

```diff
@@ -119,7 +120,8 @@ def refractionCoefficients(wavelength=0.5, site=None):
     precision = 1.e-10
 
     if site is None:
-        raise RuntimeError("Cannot call refractionCoefficients; no site information")
+        raise RuntimeError(
+            "Cannot call refractionCoefficients; no site information")
```

Comment:
Sims uses this as well. I think perhaps I had autopep8 configured incorrectly (I had it set up to use a config file, but I think perhaps the config file had an extra line that didn't quite work). I did see a lot of weird line splittings even then.